aries-site/src/layouts/content/InPageNavigation.js

```diff
@@ -133,9 +134,9 @@ export const InPageNavigation = ({ title }) => {
                       {headingTitle}
                     </Text>
                   </Box>
-                </Button>
-              </Link>
-            </Box>
+                </Box>
```

Comment:
Just wondering, can we combine these 2 Box components into 1? Everything else besides that looks good!